packages/next-app/src/app/internalmanager/(components)/TokenTable.tsx

```diff
@@ -34,32 +39,29 @@ export function TokenTable() {
   const tokensWithBalance = internalBalanceData?.user?.userInternalBalances;
 
   return (
-    <div className="h-full flex-1 flex w-full justify-center text-white">
-      <div className="mt-10">
-        {tokensWithBalance && tokensWithBalance?.length > 0 && (
-          <Table>
-            <Table.HeaderRow>
-              <Table.HeaderCell>Token Logo</Table.HeaderCell>
-              <Table.HeaderCell>Symbol</Table.HeaderCell>
-              <Table.HeaderCell>Address</Table.HeaderCell>
-              <Table.HeaderCell>Balance</Table.HeaderCell>
-              <Table.HeaderCell>
-                <span className="sr-only">Withdraw</span>
-              </Table.HeaderCell>
-            </Table.HeaderRow>
-            <Table.Body>
-              {tokensWithBalance.map((token) => (
-                <TableRow
-                  key={token.tokenInfo.address}
-                  token={token}
-                  chainId={chain!.id?.toString?.()}
-                  userAddress={addressLower as `0x${string}`}
-                />
-              ))}
-            </Table.Body>
-          </Table>
-        )}
-      </div>
+    <div className="flex-1 flex w-full text-white justify-center">
+      {tokensWithBalance && tokensWithBalance?.length > 0 && (
+        <Table>
+          <Table.HeaderRow>
+            <Table.HeaderCell>
+              <span className="sr-only">Token Logo</span>
```

Comment:
this 1st column (token logo) is a bit large, could you decrease the width a bit?